src/components/container/ConnectDevicesMenu/ConnectDevicesMenu.tsx

```diff
@@ -149,19 +162,25 @@ export default function (props: ConnectDevicesMenuProps) {
             return null;
         }
 
-        return <AppleHealthMenuItem platform={platform!} />;
+        return <AppleHealthMenuItem preview={!!props.previewState} platform={platform!} />;
     }
 
     function getGoogleFitMenuItem() {
         if (!accountTypes.includes("GoogleFit") || platform == "iOS") {
             return null;
         }
 
-        let action = () => MyDataHelps.showGoogleFitSettings();
+        let action = () => {
+            if ( props.previewState ) return;
+            MyDataHelps.showGoogleFitSettings();
+        };
         let indicator = <div className="mdhui-connect-devices-menu-connect">{language("settings")}</div>;
 
         if (platform == "Web") {
-            action = () => MyDataHelps.openExternalUrl("https://play.google.com/store/apps/details?id=com.careevolution.mydatahelps&hl=en_US&gl=US");
+            action = () => {
+                if ( props.previewState ) return;
+                MyDataHelps.openExternalUrl("https://play.google.com/store/apps/details?id=com.careevolution.mydatahelps&hl=en_US&gl=US");
```

Comment:
Minor suggestion: you added this `if ( props.previewState ) return;` pattern in a bunch of places, it could be a `handleMenuItemClick( myActionHere )` type of utility.

Reply:
Done.